examples/resources/ec_waf_scopes/resource.tf

```diff
@@ -0,0 +1,99 @@
+resource "ec_waf_scopes" "scopes1" {
+  account_number = var.account_number
+
+  scope {
+    name           = "Scopes Test"
+    
+    host {
+      is_case_insensitive = false
+      type                = "EM"
+      values              = ["site1.com/path1","site2.com"]
+    }
+
+    limit {
+      id                   = "<Rate Rule ID #1>"
```

Comment:
Is it generated by server?
If so, no need to be here?

Reply:
"Limit" refers to Rate rules, so this is a Rate Rule ID. So it is already a separate resource. The naming is just not consistent on the WAF API side.